sdk/sdktypes/id.go

```diff
@@ -73,6 +76,14 @@ func (i id[T]) Value() string {
 	return i.tid.Suffix()
 }
 
+func (i id[T]) UUIDValue() UUID {
+	if !i.IsValid() {
+		return UUID{}
+	}
+
+	return uuid.UUID(i.tid.UUIDBytes())
+}
+
```

Comment:
i think this is nicer

```suggestion
func (i id[T]) UUIDValue() (uuid UUID) {
	if i.IsValid() {
	    uuid = i.tid.UUIDBytes()
	}
	return
}

```